xilinx_test/kernels/HBMandBlockCopy.cpp

```diff
@@ -0,0 +1,11 @@
+#define DATA_SIZE 1024
```

Comment:
Macros are for C-code, in C++ we have constexpr:
```suggestion
constexpr int kDataSize = 1024;
```